proto/docs/p4runtime.md

```diff
@@ -475,9 +475,12 @@ action.
 
 ### Clone Session Entry
 
-A clone session entry is a tuple consisting of a clone session id, egress port
-and clone class of service. As above, all fields are of type *uint64* to
-accomodate bitwidths up to 64 for the corresponding PSA metadata fields.
+A clone session entry is a tuple consisting of a clone session id, egress port,
+clone class of service and packet length. The packet length (bytes) must be set
+to a non-zero value if truncation of the clone is required. Otherwise, the
+packet length is unspecified (defaults to 0). As above, all fields are of type
```

Comment:
This is a little confusing to me: shouldn't the default (packet length of 0) mean no truncation for the cloned packet?

Reply:
Ya, I think that is what I am saying. Will word it better.

Reply:
Oh I see, sorry I read too fast. It's the sentence "Otherwise, the packet length is unspecified (defaults to 0)" that confused me. The rest of the text is clear. I'll approve the PR but I think this one sentence should be improved.